missionbit/settings.py

```diff
@@ -194,3 +193,11 @@
 LOGIN_REDIRECT_URL = "home-home"
 
 CRISPY_TEMPLATE_PACK = "bootstrap4"
+
+GROUP_IDS = {
+              "student": 1,
+              "staff": 2,
+              "donor": 3,
+              "teacher": 4,
+              "volunteer": 5
+            }
```

Comment:
This probably belongs in a model file rather than as a setting. It wouldn't make sense to change this.

Reply:
Can we just use this to query the already existing Group model?
`forms = Form.objects.filter(recipient_groups=Group.objects.get("staff").id)`

Reply:
Fixed in bf71b0e83ec58eefa17357ea2f7b68d862fd4a10 and ea74c2a405925ab682d4fcf0f6b25917494b9a35 (model query error in first of two commits)